### PR TITLE
Add docker-compose.yml

### PR DESCRIPTION
**Pull request by bgpat**
## WHY

Run private godoc server with `docker-compose`.

## WHAT

Add `docker-compose.yml` to run gddo-server and redis.
If use private repositories on github.com, you need set your access token.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    #build: .
+    depends_on:
+      - redis
+    ports:
+      - "8080:8080"
+    environment:
+      REDIS_URL: redis://redis:6379
+      GITHUB_TOKEN: "your access token for github.com"
+  redis:
+    image: redis
```
Comment (dtan4) [inline docker-compose.yml]: Please specify image tag.

Comment (bgpat) [inline docker-compose.yml]: Fixed it.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  server:
+    image: quay.io/wantedly/godoc:latest
+    #build: .
+    depends_on:
+      - redis
+    ports:
+      - "8080:8080"
+    environment:
+      REDIS_URL: redis://redis:6379
+      GITHUB_TOKEN: "your access token for github.com"
```
Comment (dtan4) [inline docker-compose.yml]: If it assumes that `GITHUB_TOKEN` will be always rewritten by user , please consider to using `env_file` and `.env`.

```bash
$ cat .env.sample
REDIS_URL=redis://redis:6379
GITHUB_TOKEN=
$ cp .env.sample .env
$ vim .env # fill GITHUB_TOKEN
$ docker-compose up
```

Comment (bgpat) [inline docker-compose.yml]: Fixed. Please review.

**Comment by dtan4** [comment]
Please add `.env` to `.gitignore`.

**Comment by bgpat** [comment]
Added `.env` to `.gitignore`.

**Comment by dtan4** [comment]
LGTM 👍

**Comment by koudaiii** [comment]
LGTM!!

**Pull request merged**